doc/reference/dnspolicy.md

```diff
@@ -19,45 +19,35 @@
 
 ## DNSPolicySpec
 
-| **Field**         | **Type**                                                                                                                                    |  **Required**  | **Description**                                                |
-|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------|:--------------:|----------------------------------------------------------------|
-| `targetRef`       | [Gateway API PolicyTargetReference](https://gateway-api.sigs.k8s.io/geps/gep-713/?h=policytargetreference#policy-targetref-api)    |      Yes       | Reference to a Kuberentes resource that the policy attaches to |
-| `healthCheck`     | [HealthCheckSpec](#healthcheckspec)                                                                                                         |       No       | HealthCheck spec                                               |
-| `loadBalancing`   | [LoadBalancingSpec](#loadbalancingspec)                                                                                                     |       No       | LoadBancking Spec                                              |
-| `routingStrategy` | String (immutable)                                                                                                                                     |      Yes       | **Immutable!** Routing Strategy to use, one of "simple" or "loadbalacned"     |
+| **Field**         | **Type**                                                                                                                          | **Required** | **Description**                                                           |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------|:------------:|---------------------------------------------------------------------------|
+| `targetRef`       | [Gateway API PolicyTargetReference](https://gateway-api.sigs.k8s.io/geps/gep-713/?h=policytargetreference#policy-targetref-api)   |     Yes      | Reference to a Kubernetes resource that the policy attaches to            |
+| `healthCheck`     | [HealthCheckSpec](#healthcheckspec)                                                                                               |      No      | HealthCheck spec                                                          |
+| `loadBalancing`   | [LoadBalancingSpec](#loadbalancingspec)                                                                                           |      No      | LoadBalancing Spec                                                        |
```

Comment:
It should be probably be Required as loadbalancing.geo and loadbalancing.weight are required

Reply:
@pehala can you create an issue for that where it can be discussed

Reply:
I think it is just a small addition to this doc, loadbalancing section is required when strategy is loadbalanced, I would just mention it here